client/my-sites/site-settings/form-general.jsx

```diff
@@ -499,6 +500,8 @@ const FormGeneral = React.createClass( {
 					</form>
 				</Card>
 
+				{ this.props.site.jetpack && <JetpackSyncPanel siteId={ this.props.site.ID } /> }
```

Comment:
I think we should also have a Jetpack version check here Since Jetpack version older then 4.2 would not have this feature.